descriptors/ModuleDescriptor-template.json

```diff
@@ -109,6 +109,18 @@
         }
       ]
     },
+    {
+      "id": "voucher",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
```

Comment:
can be added to the existing "voucher" interface

Reply:
Done

Reply:
Done